Jenkinsfile

```diff
@@ -94,9 +95,25 @@ def buildGrakn() {
     sh "build-grakn.sh ${env.BRANCH_NAME}"
 }
 
+//Run all tests
+node {
+    String workspace = pwd()
+    checkout scm
+
+    slackGithub "Janus tests started"
+
+    timeout(90) {
+	stage('Run Janus test profile') {
+	    sh "mvn clean verify -P janus -U -Djetty.log.level=WARNING -Djetty.log.appender=STDOUT || true"
+	    junit "**/target/surefire-reports/*.xml"
```

Comment:
We think this means if the tests fail, the build won't. I guess we need a `try...finally` instead.